Bottom: 501244f52681273979e3ea577e635987e94396ef
Top:    00f6bf73bb3863e6a54d7b9c01bdc24f60cab230
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 22:42:13 +0100

Refresh of misc-c-remove-unused-delta-arg

---

diff --git a/src/select.c b/src/select.c
index e42d4bb..8dfe270 100644
--- a/src/select.c
+++ b/src/select.c
@@ -87,7 +87,7 @@ SelectPin (LibraryEntryTypePtr entry, bool toggle)
 	    if (toggle)
 	      {
 		TOGGLE_FLAG (SELECTEDFLAG, pin);
-		CenterDisplay (pin->X, pin->Y, false);
+		CenterDisplay (pin->X, pin->Y);
 	      }
 	    else
 	      SET_FLAG (SELECTEDFLAG, pin);
@@ -103,7 +103,7 @@ SelectPin (LibraryEntryTypePtr entry, bool toggle)
 	    if (toggle)
 	      {
 		TOGGLE_FLAG (SELECTEDFLAG, pad);
-		CenterDisplay (pad->Point1.X, pad->Point1.Y, false);
+		CenterDisplay (pad->Point1.X, pad->Point1.Y);
 	      }
 	    else
 	      SET_FLAG (SELECTEDFLAG, pad);
